### DISCUSSION TITLE
CSS Battle #1 – Simply Square

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/virtualcoffee.io#886] [CSS Battles]
### Link to battle:

[Let's battle! ⚔️](https://cssbattle.dev/play/1)

### Copy the code block below to format your comment on the discussion thread:

````md
<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>
<!-- have to be followed by an empty line! -->

```html
your cssbattles code here!
```

</details>
````

### What others will see:

This will result in a nice hidden bit like so:

<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>

```html
your cssbattles code here!
```
</details>

**Comment by meg-gutshall** [comment]
<details> 
<summary>First round  – <strong>691.3 {155}</strong></summary>

```html
<div></div>
<style>
  body {
    margin: 0;
    background: #5d3a3a;
  }
  div {
    width: 200px;
    height: 200px;
    background: #b5e0ba;
  }
</style>
```
</details>

<details> 
<summary>Refactored using <code>linear-gradient</code>  – <strong>709.43 {136}</strong></summary>

```html
<style>body { background: linear-gradient(transparent 200px, #5d3a3a 200px), linear-gradient(90deg, #b5e0ba 50%, #5d3a3a 50%); }</style>
```
</details>

<details> 
<summary>Then minified  – <strong>731.15 {117}</strong></summary>

```html
<style>body{background:linear-gradient(transparent 200px,#5d3a3a 200px),linear-gradient(90deg,#b5e0ba 50%,#5d3a3a 50%
```
</details>

<details>
<summary>Using inline styling  – <strong>776.23 {86}</strong></summary>


```html
<body bgcolor=#5d3a3a style=margin:0><p style=background:#b5e0ba;width:200;height:200>
```

</details>

**Reply by meg-gutshall** [reply]
<details>
<summary>I'm getting closer... – <strong>799.48 {73}</strong></summary>


```html
<style>&{background:#5d3a3a;border-left:50vw dashed#b5e0ba;margin:0 0 100
```

</details>

<details>
<summary>Caught ya, @Narigo!! – <strong>807.23 {69}</strong></summary>


```html
<style>*{background:#5d3a3a}&>*{background:#b5e0ba;margin:0 200 100 0
```

</details>

**Comment by AmyShackles** [comment]
<details>
<summary>Code Source  – <strong>771.26 {89}</strong></summary>


```html
<body style=background:#5d3a3a;margin:0><p style=background:#b5e0ba;width:200;height:200>
```

</details>

**Comment by Narigo** [comment]
<details>
<summary>Code Source  – <strong>807.23 {69}</strong></summary>


```html
<style>*>*{height:384;background:conic-gradient(#5d3a3a 75%,#b5e0ba 0
```

</details>
